numpy/ma/core.py

```diff
@@ -3105,6 +3105,13 @@ def __getitem__(self, indx):
         Return the item described by i, as a masked array.
 
         """
+        warnings.warn(
```

Comment:
Might add a note before the warning, here and below, with the date and release of the warning. That will help us keep track of how much time has passed since. Something like
```
# 01/15/2016 1.11.0
```
would do.